docs/api/js/accounts-l1-l2.md

```diff
@@ -47,8 +47,8 @@ import { ethers } from "ethers";
 
 const PRIVATE_KEY = "<WALLET_PRIVATE_KEY>";
 
-const zkSyncProvider = new Provider("https://testnet.era.zksync.dev/");
-const ethereumProvider = ethers.getDefaultProvider("goerli");
+const zkSyncProvider = new Provider("https://sepolia.era.zksync.dev");
+const ethereumProvider = ethers.getDefaultProvider("sepolia");
```

Comment:
@idea404 methods like `getDefaultProvider` are what will need to be updated to support this action fyi. These changes need to coincide with Sepolia support. To be clear this example, is for an Ethereum provider so its fine but we also have this method in our sdk.

Reply:
zksync2-js [`0.4.0-beta.2`](https://www.npmjs.com/package/zksync2-js?activeTab=versions) [supports this](https://github.com/zksync-sdk/zksync2-js/commit/cfcd9dbb9c6437cf9707f6ea793a6cf13e402a07)